app/src/main/kotlin/com/rockspin/bargainbits/data/models/GroupedGameDeal.kt

```diff
@@ -0,0 +1,30 @@
+package com.rockspin.bargainbits.data.models
+
+import java.util.*
```

Comment:
This is super weird - the Kotlin auto complete includes the whole util package, not just Date. I can manually fix it in this case but i don't think i'll pay attention when I auto import in future.